toqito/state_opt/state_exclusion.py

```diff
@@ -92,6 +134,13 @@ def state_exclusion(
         `float` output to a certain amount of precision such that the value deviates after a few digits
         of accuracy.
 
+    .. note::
+        If you encounter a `ZeroDivisionError` or an `ArithmeticError` when using cvxopt as a solver (which is the
+        default), you might want to set the `abs_ipm_opt_tol` option to a lower value (the default being `1e-8`) or
+        to set the `cvxopt_kktsolver` option to `ldl`.
+
+        See https://gitlab.com/picos-api/picos/-/issues/341
```

Comment:
If you are adding a link as a reference, add it to `docs/articles.bib` and then cite it. Look for how we add Wikipedia links in this file.

Reply:
That's how I added one such issue: https://github.com/vprusso/toqito/blob/49ecfd489d8be0fea23f88291779a2ef03065d23/docs/articles.bib#L217

Reply:
Gotcha. My preference would be to keep these types of things in line, as the issues are somewhat ephemeral once they are closed out. 

As an aside, (and this is likely a separate issue that I should put on the board) but I think it makes sense to consolidate the references into a single `.bib` file. Not that `mitiq` needs to be the gold standard, but in seeing how they arrange things, it seems like a single `refs.bib` is used and I don't see any gitlab/github URLs located in those reference files. 

I can see an argument for keeping the URL in the bib file, but my vote would be to keep them out.